examples/default.yaml

```diff
@@ -39,6 +39,10 @@ memory: null
 # 🟢 Builtin default: "100GiB"
 disk: null
 
+# Data size
```

Comment:
I think this needs additional documentation to explain that this is a supplemental data volume, and the parameter is the size, and can use size-related units like "GiB". Maybe the builtin default should change to "0GiB" to make this obvious?

Reply:
The `datadisk` also needs to be mentioned in `docs/internal.md`.